Dockerfile.win10.min

```diff
@@ -26,7 +26,7 @@
 
 # Windows min container for Triton build
 
-ARG BASE_IMAGE=mcr.microsoft.com/dotnet/framework/sdk:4.8
+ARG BASE_IMAGE=mcr.microsoft.com/windows:10.0.19042.1889
```

Comment:
Can you share more background on this change? Is there a ticket for this?

Reply:
Hi @Tabrizian , 
here is a related issue for it:
https://github.com/triton-inference-server/server/issues/4981